### PR TITLE
Fix code formatting and update copyright legal name

### PR DESCRIPTION
**Pull request by jbakerTT**
## ⚠️ CODEOWNERS REVIEW REQUIRED

**These files require codeowner review and sign-off before merging.**

### Background
During our update of SPDX copyright headers to reflect the correct legal entity name (`Tenstorrent USA, Inc.`), these 17 files failed the Black Python formatter pre-commit checks. They were originally excluded from PR #41407 to allow that PR to merge cleanly.

### What This PR Does
1. Updates SPDX copyright headers from `Tenstorrent Inc.` to `Tenstorrent USA, Inc.`
2. Applies Black formatting fixes to pass repository pre-commit standards

### Files Modified (Experimental Model Reference Implementations)
- `models/experimental/blazepose/demo/blazebase.py`
- `models/experimental/distilbert/demo/cpu_demo.py`
- `models/experimental/efficientnet/reference/efficientnet_lite.py`
- `models/experimental/hrnet/demo/cpu_demo.py`
- `models/experimental/hrnet/reference/hrnet.py`
- `models/experimental/inceptionV4/reference/_utils.py`
- `models/experimental/inceptionV4/reference/inception.py`
- `models/experimental/llama2/llama2_utils.py`
- `models/experimental/nanogpt/reference/bench.py`
- `models/experimental/nanogpt/reference/model.py`
- `models/experimental/nanogpt/reference/sample.py`
- `models/experimental/squeezebert/demo/cpu_demo.py`
- `models/experimental/squeezebert/squeezebert_utils.py`
- `models/experimental/ssd/demo/cpu_demo.py`
- `models/experimental/ssd/reference/backbone.py`
- `models/experimental/ssd/reference/ssd.py`
- `models/experimental/ssd/reference/ssd_head.py`

### Action Required from CODEOWNERS
**Please verify that the Black formatting changes do not alter the functional behavior of these reference implementations.** These are typically reference code from external sources, and while the formatting changes are necessary to meet repository standards, we need confirmation that the code will still work as intended.

Once approved, this will complete the copyright name change initiative.

### Related
- Follows PR #41407 (Legal: Update copyright for files added/modified after PR #41389)
- Part of the repository-wide copyright update to `Tenstorrent USA, Inc.`

### CODE REVIEW

File: models/experimental/ssd/demo/cpu_demo.py [hunk 1]
```diff
@@ -16,9 +16,7 @@ def test_cpu_demo():
     image = transforms.ToTensor()(image).unsqueeze(0)
 
     with torch.no_grad():
-        torch_ssd = ssdlite320_mobilenet_v3_large(
-            weights=SSDLite320_MobileNet_V3_Large_Weights
-        )
+        torch_ssd = ssdlite320_mobilenet_v3_large(weights=SSDLite320_MobileNet_V3_Large_Weights)
```
[the remark this answers is not in the document]
Comment (jbakerTT) [reply models/experimental/ssd/demo/cpu_demo.py:19]: 📋 **Codeowner attention needed**

@dgomezTT @jmalone-tt @ayerofieiev-tt @uaydonat - Copilot has flagged a potential runtime issue with the weights parameter in this SSD demo. Please review this suggestion and determine if it should be addressed.

File: models/experimental/ssd/reference/ssd_head.py [hunk 2]
```diff
@@ -166,23 +156,13 @@ def _mobilenet_extractor(
...
     if not 0 <= trainable_layers <= num_stages:
-        raise ValueError(
-            "trainable_layers should be in the range [0, {num_stages}], instead got {trainable_layers}"
-        )
-    freeze_before = (
-        len(backbone)
-        if trainable_layers == 0
-        else stage_indices[num_stages - trainable_layers]
-    )
+        raise ValueError("trainable_layers should be in the range [0, {num_stages}], instead got {trainable_layers}")
```
[the remark this answers is not in the document]
Comment (jbakerTT) [reply models/experimental/ssd/reference/ssd_head.py:164]: 📋 **Codeowner attention needed**

@dgomezTT @jmalone-tt @ayerofieiev-tt @uaydonat - Copilot has identified that this error message has placeholders but isn't using an f-string, so the variable values won't be displayed. Please review this suggestion and determine if it should be addressed.

File: models/experimental/inceptionV4/reference/_utils.py [hunk 3]
```diff
@@ -127,9 +115,7 @@ def assign_weight_batchnorm(norm: nn.BatchNorm2d, state_dict, key_w: str):
     norm.bias = nn.Parameter(state_dict[f"{key_w}.bias"])
     norm.running_mean = nn.Parameter(state_dict[f"{key_w}.running_mean"])
     norm.running_var = nn.Parameter(state_dict[f"{key_w}.running_var"])
-    norm.num_batches_tracked = nn.Parameter(
-        state_dict[f"{key_w}.num_batches_tracked"], requires_grad=False
-    )
+    norm.num_batches_tracked = nn.Parameter(state_dict[f"{key_w}.num_batches_tracked"], requires_grad=False)
     norm.eval()
```
[the remark this answers is not in the document]
Comment (jbakerTT) [reply models/experimental/inceptionV4/reference/_utils.py:119]: 📋 **Codeowner attention needed**

@dgomezTT @jmalone-tt @ayerofieiev-tt @uaydonat - Copilot has flagged a potential issue with how BatchNorm2d buffers are being handled. The suggestion is to use buffer assignment methods rather than converting to parameters. Please review this suggestion and determine if it should be addressed.
